README.md

```diff
@@ -60,6 +62,10 @@ Add the labels `foo` and `bar` only to open issues in repository `sshaw/git-link
 
     batchlabels -i add foo bar sshaw/git-link
 
+To add labels having more than one word, use backslash `\` to escape spaces. Example below will add `good first issue` to all open issues in the repository `sshaw/git-link:
+
+    batchlabels -i add good\ first\ issue sshaw/git-link
```

Comment:
Quoting should work as well: `add 'good first issue'`

Reply:
Thanks, that's great. You should push another example or add on this one as people would find it easier to do just that.